Formula/httest.rb

```diff
@@ -16,17 +16,30 @@ class Httest < Formula
   depends_on "openssl"
   depends_on "pcre"
   depends_on "lua"
+  depends_on "nghttp2"
+  depends_on "spidermonkey"
 
   def install
     # Fix "fatal error: 'pcre/pcre.h' file not found"
     # Reported 9 Mar 2017 https://sourceforge.net/p/htt/tickets/4/
     (buildpath/"brew_include").install_symlink Formula["pcre"].opt_include => "pcre"
     ENV.prepend "CPPFLAGS", "-I#{buildpath}/brew_include"
 
+    # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for libxml2
+    if MacOS.version == :sierra || MacOS.version == :el_capitan
+      ENV["SDKROOT"] = MacOS.sdk_path
+    end
+
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
+                          "--enable-lua-module",
+                          "--enable-html-module",
+                          "--enable-xml-module",
+                          "--enable-h2-module",
+                          "--enable-js-module",
                           "--with-apr=#{Formula["apr"].opt_bin}",
-                          "--enable-lua-module"
+                          "--with-nghttp2=#{Formula["nghttp2"].opt_bin}",
```

Comment:
does it not find this and spidermonkey on its own automatically?

Reply:
also are you sure you mean opt_bin not opt_prefix?

Reply:
sure, will try without explicit references.